genePred/testGenePred.gpd

```diff
@@ -0,0 +1,2 @@
+test	0	+	0	1000	0	900	0,18,500,800,	2,20,502,802,
```

Comment:
I think this should have a "4" in a column after the "900" column to match the UCSC format.  See the file the comes out of the table browser, or this table, https://genome.ucsc.edu/FAQ/FAQformat.html#format9 , where each element in the table is the next column.